Source/ProgressionSystemRuntime/Private/Data/PSWorldSubsystem.cpp

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#include "Data/PSWorldSubsystem.h"
+
+#include "Controllers/MyPlayerController.h"
+#include "Data/PSDataAsset.h"
+#include "Data/PSSaveGameData.h"
+#include "Kismet/GameplayStatics.h"
+#include "LevelActors/PlayerCharacter.h"
+#include "MyDataTable/MyDataTable.h"
+#include "MyUtilsLibraries/UtilsLibrary.h"
+#include "UtilityLibraries/MyBlueprintFunctionLibrary.h"
+
+#include UE_INLINE_GENERATED_CPP_BY_NAME(PSWorldSubsystem)
+
+// Returns this Subsystem, is checked and wil crash if can't be obtained
+UPSWorldSubsystem& UPSWorldSubsystem::Get()
+{
+	const UWorld* World = UUtilsLibrary::GetPlayWorld();
+	checkf(World, TEXT("%s: 'World' is null"), *FString(__FUNCTION__));
```

Comment:
use call `World->`, however never included it. The same for GEngine. It will fail the cook.
Please add:
```
#include "Engine/Engine.h"
#include "Engine/World.h"
```

Reply:
added